pmdarima/__init__.py

```diff
@@ -22,7 +22,7 @@
 # Dev branch marker is: 'X.Y.dev' or 'X.Y.devN' where N is an integer.
 # 'X.Y.dev0' is the canonical version of 'X.Y.dev'
 #
-__version__ = "1.3.1.dev0"
+__version__ = "1.4.0.dev0"
```

Comment:
If we want to cut 1.4.0 this weekend, do we want to drop the `dev` tag?

Reply:
yeah but not in this pr